docs/ref/checks.txt

```diff
@@ -96,7 +96,8 @@ Backwards compatibility
 -----------------------
 
 The following checks are performed to warn the user of any potential problems
-that might occur as a result of a version upgrade.
+that might occur as a result of a version upgrade. Some have been removed since
+they were added as per the deprecation timeline.
```

Comment:
I'm not sure if this will be helpful. Readers may not know what the deprecation timeline is and even if they do, the timeline doesn't discuss particulars of what checks are removed when. Maybe it's better just to remove old compatibility checks from the documentation when they're removed. I think the main reason to leave the docs for other removed checks is so that their code isn't reused. That's not an issue here since the code is based on the version number.

Reply:
So, keep the heading and replace all content with a paragraph like "Sometimes checks are added for backwards compatibility between Django versions. At current there are no such checks, as they've been removed since they were no longer needed." ?

Reply:
Yes, done in f8946fbb5d7e8eb6c8ca7481b0cd4e78627ef8cf.